build_matrix

```diff
@@ -3,13 +3,11 @@
 import argparse
 import json
 import os
-from urllib.request import urlopen
 from pprint import pprint
 
 
-def main(matrix_url, include, exclude):
-    with urlopen(matrix_url) as response:
-        matrix = json.load(response)
+def main(matrix, include, exclude):
+    matrix = json.load(matrix)
```

Comment:
the one matrix is a string, the other a dict, someone gonna yell at you (→ mypy).

Reply:
That's what you get for renaming variables.